ecosystem/sep-0018.md

```diff
@@ -50,6 +50,20 @@ wallet.eth
 * **Term REGEXP:** `[_a-z][_a-z0-9]*`
 * **Key REGEXP:** `[_a-z][_a-z0-9]*(\.[_a-z][_a-z0-9]*)*`
 
+## Namespace Registry
+
+To guarantee interoperability and avoid naming conflicts, namespaces must be
+used as defined in SEP documents.
+
+- `app`: Space dedicated to vendor-specific keys. For example, "example.org" can
+  define its keys under `app.example_org`.
+- `config`: Space dedicated to ecosystem-wide configuration. Those are values
+  that impact software logic.
+  - `config.memo_required`: [SEP29](sep-0029.md)
+  - `config.multisig`: [SEP19](sep-0019.md)
+- `profile`: Space dedicated to account identification.
+- `wallet`: Space dedicated to external ledger addresses.
```

Comment:
Are there wallets or apps that are using these values today?

If not I think we should hold off adding namespaces until there are SEPs or use cases that are going to use them.

If so, could you point to some examples so it's easier to understand how they're using data entries and these fields?

Reply:
The point is precisely to prevent naming conflict by providing indication of how applications should use those namespaces (that is, putting vendor stuff under `app`). If we wait for them to implement their own things it's pointless.

Reply:
Got it, that makes sense. I think that makes sense for `app`, I have some more questions about `profile` and `wallet` I'll post in a new thread.